src/gigs-board/pages/Create.jsx

```diff
@@ -0,0 +1,537 @@
+const existingLabelSet = new Set(existingLabelStrings);
+const existingLabels = existingLabelStrings.map((s) => {
+  return { name: s };
+});
+
+const labelEditor = (
+  <div className="col-lg-12 mb-2">
+    <p className="fs-6 fw-bold mb-1">Labels</p>
+    <Typeahead
+      multiple
+      labelKey="name"
+      onInputChange={checkLabel}
+      onChange={setLabels}
+      options={existingLabels}
+      placeholder="near.social, widget, NEP, standard, protocol, tool"
+      selected={state.labels}
+      positionFixed
+      allowNew={(results, props) => {
+        return (
+          !existingLabelSet.has(props.text) &&
+          props.selected.filter((selected) => selected.name === props.text)
+            .length == 0 &&
+          Near.view(
+            nearDevGovGigsContractAccountId,
+            "is_allowed_to_use_labels",
+            { editor: context.accountId, labels: [props.text] }
+          )
+        );
+      }}
+    />
+  </div>
+);
+
+const nameDiv = (
+  <div className="col-lg-6 mb-2">
+    <p className="fs-6 fw-bold mb-1">Title</p>
+    <input
+      type="text"
+      value={state.name}
+      onChange={(event) => State.update({ name: event.target.value })}
+    />
+  </div>
+);
+
+const descriptionDiv = (
+  <div className="col-lg-12 mb-2">
+    <p className="fs-6 fw-bold mb-1">Description</p>
+    <textarea
+      value={state.description}
+      type="text"
+      rows={6}
+      className="form-control"
+      onChange={(event) => State.update({ description: event.target.value })}
+    />
+  </div>
+);
+
+const isFundraisingDiv = (
+  // This is jank with just btns and not radios. But the radios were glitchy af
+  <>
+    <div class="mb-2">
+      <p class="fs-6 fw-bold mb-1">
+        Are you seeking funding for your solution?
+        <span class="text-muted fw-normal">(Optional)</span>
+      </p>
+      <div class="form-check form-check-inline">
+        <button
+          className="btn btn-light p-0"
+          style={{
+            backgroundColor: state.seekingFunding ? "#0C7283" : "inherit",
+            color: "#f3f3f3",
+            border: "solid #D9D9D9",
+            borderRadius: "100%",
+            height: "20px",
+            width: "20px",
+          }}
+          onClick={() => State.update({ seekingFunding: true })}
+        ></button>
+        <label class="form-check-label" for="inlineRadio1">
+          Yes
+        </label>
+      </div>
+      <div class="form-check form-check-inline">
+        <button
+          className="btn btn-light p-0"
+          style={{
+            backgroundColor: !state.seekingFunding ? "#0C7283" : "inherit",
+            color: "#f3f3f3",
+            border: "solid #D9D9D9",
+            borderRadius: "100%",
+            height: "20px",
+            width: "20px",
+          }}
+          onClick={() => State.update({ seekingFunding: false })}
+        ></button>
+        <label class="form-check-label" for="inlineRadio2">
+          No
+        </label>
```

Comment:
With this markup, clicking on "Yes" or "No" will select the appropriate "radio-button".

```suggestion
        <label class="form-check-label">
          <button
            className="btn btn-light p-0"
            style={{
              backgroundColor: state.seekingFunding ? "#0C7283" : "inherit",
              color: "#f3f3f3",
              border: "solid #D9D9D9",
              borderRadius: "100%",
              height: "20px",
              width: "20px",
            }}
            onClick={() => State.update({ seekingFunding: true })}
          />
          Yes
        </label>
      </div>
      <div class="form-check form-check-inline">
        <label class="form-check-label">
          <button
            className="btn btn-light p-0"
            style={{
              backgroundColor: !state.seekingFunding ? "#0C7283" : "inherit",
              color: "#f3f3f3",
              border: "solid #D9D9D9",
              borderRadius: "100%",
              height: "20px",
              width: "20px",
            }}
            onClick={() => State.update({ seekingFunding: false })}
          />
          No
        </label>
```